packages/Button/src/LinkButton.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+// import iconButtonStyles from "./IconButton.styles";
+import Button from "./Button";
+
+const LinkPropTypes = {
+  children: PropTypes.node.isRequired,
+  kind: PropTypes.oneOf(Button.Kinds.ALL),
+  size: PropTypes.oneOf(ShirtSizes.MEDIUM),
+  url: PropTypes.string,
```

Comment:
Besides a url prop, we want to also support if the link wants to be opened in a new tab.

If the link is opened in a new tab we need to support a security vulnerability.
Read more about it here: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
After your props are completed, please add a comment above each prop for documentation purposes.

Example: https://github.com/acl-services/paprika/blob/master/packages/Button/src/Button.js#L12